packages/database/src/migrations/20210524111255-StriveAdultTrappingReportConfig-modifies-data.js

```diff
@@ -0,0 +1,75 @@
+'use strict';
+
+import { generateId, insertObject } from '../utilities';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+const permissionGroupNameToId = async (db, name) => {
+  const record = await db.runSql(`SELECT id FROM permission_group WHERE name = '${name}'`);
+  return record.rows[0] && record.rows[0].id;
+};
+
+const REPORT = {
+  id: generateId(),
+  code: 'PG_Strive_Mosquito_Species_Distribution_AE_AT',
+  config: {
+    fetch: {
+      aggregations: [
+        {
+          type: 'RAW',
+          config: {
+            dataSourceEntityType: 'facility',
+          },
+        },
+      ],
+      dataElements: ['STRVEC_AE-AT13'],
+    },
+    transform: [
+      {
+        transform: 'filter',
+        where: "exists($row['value'],$row['dataElement'])",
```

Comment:
Does `exists()` support multiple arguments like this? Perhaps it should... could be good...

Regardless, do we need this transform? I think it's safe to assume that every analytic has a `value` and a `dataElement` 👍

Reply:
happy to drop this.